docs/diagnostics/PX1033.md

```diff
@@ -5,12 +5,16 @@ This document describes the PX1033 diagnostic.
 
 | Code   | Short Description                                 | Type  | Code Fix    | 
 | ------ | ------------------------------------------------- | ----- | ----------- | 
-| PX1033 | The DAC does not have an explicit primary key declaration | Warning (ISV Level 3: Informational) | Unavailable | 
+| PX1033 | The DAC does not have an explicit primary key declaration | Warning (ISV Level 3: Informational) | Available | 
 
 ## Diagnostic Description
-You can define primary key in DACs. The diagnostic suggests adding primary key definition to a DAC. The warning is shown only for DACs that have either PXCacheName or PXPrimaryGraph attribute.
+You can define a primary key field in a DAC. The diagnostic suggests adding a primary key definition to a DAC. 
+The warning is shown only for DACs that have the `PXCacheName` or `PXPrimaryGraph` attribute.
+The diagnostic does not check abstract DACs and DACs which have only unbound DAC properties (so-called "virtual" DACs used for custom popups and filters for inquiry forms).
 
-## Example of Code For Which the Warning Is Shown
+The code fix generates a primary key and the 'Find' method which makes PK more useful.
```

Comment:
I would rephrase a bit - "which allows to easily find DAC row by providing values composing primary key ".